docs/local-development.md

```diff
@@ -29,6 +29,18 @@ cd boson-protocol-contracts
 npm install
 ```
 
+### Prepare git pre-hooks
+Pre-hooks are scripts that runs automatically before each [git hook](https://git-scm.com/docs/githooks) execution. 
+We use the pre-commit hook to ensure that all new code follows the style guidelines defined in [eslint](../.eslintrc.json) and [prettier](../.prettierrc) configuration files before it is committed to the repository.
+The pre-commit hook is also used to verify and fix [natspec interfaces ids](./tasks.md#verify-natspec-interface-ids)
+
+The [Husky](https://typicode.github.io/husky) library is used to manage pre-hooks scripts.
+To make husky work, you must run:
+
+```
+husky install
```

Comment:
That didn't work for me, I had to run it with

```npx husky install```

Reply:
fixed